trunk/NDHMS/MPP/Makefile

```diff
@@ -5,9 +5,17 @@
 
 include ../macros
 
-OBJS =  CPL_WRF.o mpp_land.o module_mpp_ReachLS.o module_mpp_GWBUCKET.o
+OBJS =  hashtable.o CPL_WRF.o mpp_land.o module_mpp_ReachLS.o module_mpp_GWBUCKET.o
```

Comment:
We should probably make the updates to the CMake files as well.

Reply:
Good catch; we really need to add CMake to Travis tests...